.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,31 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+  "name": "Existing Dockerfile",
+  "build": {
+    // Sets the run context to one level up instead of the .devcontainer folder.
+    "context": "..",
+    // Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+    "dockerfile": "../Dockerfile"
+  },
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/devcontainers/features/python:1": {}
+  },
+
+  // Use 'forwardPorts' to make a list of ports inside the container available locally.
+  "forwardPorts": [3000],
+
+  // Uncomment the next line to run commands after the container is created.
+  // How can I edit this to not need to reinstall the dependencies that were installed during the Dockerfile build.
+  "postCreateCommand": "pip install -r requirements.txt && ./entrypoint.sh",
```

Comment:
THANK YOU SO MUCH!
I've been waiting for an expert like you to come along and save us because yes they were killing me too. I'm AFK for now but I'll tackle this soon. You're a legend.